### DISCUSSION TITLE
UDM Base configure LAN as WAN

### DISCUSSION THREAD

**Discussion by a-tsymbal** [fabianishere/udm-kernel-tools#83] [Q&A]
Hello,

After power outage my UDM Base not able to connect to the internet.
I tried factory reset and uploading latest firmware using a recovery mode.

I'm able to setup the console using offline mode in the phone app (through a bluetooth connection).

But I'm not able to ping `192.168.1.1` when connected from a PC to the LAN1 directly.
Also when I try to connect to the console WiFi phone says `IP Configuration Failure`.

The only way to connect to the console is in **recovery mode** using `192.168.1.30` on LAN1.

In recovery mode I setup static IP on LAN3 (`eth2` - `192.168.0.32`) and WAN (`eth4` - `192.168.0.34`).
The `192.168.0.0` network is from my old router.
When I connect router LAN to console LAN3(`eth2`) I'm able to do `ping -I eth2 192.168.0.1`
When I connect router LAN to console WAN(`eth4`) I'm **not** able to do `ping -I eth4 192.168.0.1`

It is possible for other routers like [Linksys E900](https://www.youtube.com/watch?v=QLcAl0O4gG0), but there are [no OpenWrt for UDM](https://openwrt.org/toh/ubiquiti/start) only for EdgeRouter.

So I'm wondering whether it is possible to somehow reconfigure console LAN port as WAN in **recovery mode** using this tools?

**Comment by fabianishere** [comment]
I don’t believe it is possible to do that using these tools. 

However, as far as I know the WAN subnet cannot overlap with a LAN subnet (so you cannot have both be 192.168.0.0/24).

**Reply by a-tsymbal** [reply]
I'm not able to ping even when there is only one `eht4` interface configured.

But thanks for the answer anyway.
